ocs_ci/ocs/platform_nodes.py

```diff
@@ -76,6 +76,8 @@ def __init__(self):
             "powervs": IBMPowerNodes,
             "rhv": RHVNodes,
             "ibm_cloud": IBMCloud,
+            "rosa": AWSNodes,
```

Comment:
As those two are managed, I am not sure if adding nodes is handled the same like in our implementation. Please verify.